marketingcloudsdk.gemspec

```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |spec|
 
 	spec.add_dependency "savon", "2.2.0"
 	spec.add_dependency "json", "~>2.3.0"
-	spec.add_dependency "jwt", "~>1.0",">= 1.0.0"
```

Comment:
Update the GEM version as well. Lets call it 1.5.0

Reply:
Done. @mahendra-kathirvel .